### ISSUE TITLE
Create and share performance consultation process

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#26]
### Why/User Benefit/User Problem
When folks run into performance issues (e.g. need a design review, want code review, or overall feature review), they may not know they can go to our team to get help. It'd help if we:
- Had a documented way to assist other teams (e.g. a "consultation")
- Frequently reminded and encouraged folks about this process so they remember and feel comfortable reaching out to us for help

### Impact
Without this, folks may not reach out to us for performance help during development so we'll have to catch issues after the fact – i.e. it will take a longer time to fix – and we're only likely to catch issues that are in our focus areas or that are so bad we notice.

Additionally, doing this will get folks thinking about performance more regularly and get us further along in making a performance culture for our team.

### Acceptance Criteria (how do I know when I’m done?)
- Define a performance consultation process relevant to mobile
  - Leverage what has been done on desktop
- Notify the Fenix team about the process &, after bug closure), frequently remind/encourage them to use it to get more folks thinking about performance
- Notify our team about the process & remind them to encourage folks to use it: let's change the culture together!

---

Note: doing this can be useful to our WebExt investigation https://github.com/mozilla-mobile/fenix/issues/5823 because we can help out early rather than waiting for everything to be implemented (which is in large parts our only actionable plan).

---

fwiw, after discussion with Christian, I imagine performance consultation similar to a security consultation: if you were implemented a login system, you would want security to help you with the design, with code review, and doing a final security check after the fact. In fact, you probably shouldn't ship the feature until that's done. However, perf is less likely to gate releases and for each feature the PM should decide whether it's worth blocking on failing performance criteria or a lack of testing for that criteria.

**Comment by mcomella** [comment]
Triage: for speed, we should do the good thing, not the perfect thing. I have an action item to send an email to let folks know that if they want a performance consultation, they should contact me (a single point of contact for simplicity). We should especially ensure managers are aware. We should be specific about what the perf team can offer (e.g. tooling help, bugs, consultation, socializing how to identify performance problems) and note that we'll also be trying to reach out for known features, automation, relman level checks, etc.

**Comment by mcomella** [comment]
I think this email is confusing: "If you would like the front-end performance group to investigate performance issues, please contact me directly." might discourage folks from filing bugs as they do now and I don't want that. I'm inclined to close this as I think what we're doing is working well enough for GA and we can notify folks about more formal perf consultations afterwards.

Removing from project board to alleviate any confusion.

**Comment by mcomella** [comment]
Not a priority right now.
